plasmapy/formulary/parameters.py

```diff
@@ -166,77 +204,90 @@ def Alfven_speed(
     B: u.T,
     density: [u.m ** -3, u.kg / u.m ** 3],
     ion: Optional[Particle] = None,
-    z_mean=None,
+    z_mean: Optional[numbers.Real] = None,
 ) -> u.m / u.s:
     r"""
-    Return the Alfvén speed.
+    Calculate the Alfvén speed.
 
-    **Aliases:** `va_`
+    The Alfven speed :math:`V_A` is the typical propagation speed of magnetic
+    disturbances in a plasma, and is given by:
 
-    Parameters
-    ----------
-    B : ~astropy.units.Quantity
-        The magnetic field magnitude in units convertible to tesla.
+    .. math::
 
-    density : ~astropy.units.Quantity
-        Either the ion number density in units convertible to 1 / m**3,
-        or the mass density in units convertible to kg / m**3.
+        V_A = \frac{B}{\sqrt{\mu_0\rho}}
 
-    ion : ~plasmapy.particles.Particle, optional
-        Representation of the ion species (e.g., `'p'` for protons,
-        `'D+'` for deuterium, or `'He-4 +1'` for singly ionized
-        helium-4). If no charge state information is provided, then the
-        ions are assumed to be singly charged. If the density is an ion
-        number density, then this paramter is required in order to convert
-        to mass density.
+    where :math:`B` is the magnetic field and :math:`\rho = n_i m_i + n_e m_e`
+    is the total mass density (:math:`n_i` is the ion number denisty,
+    :math:`n_e` is the electron number denisty, :math:`m_i` is the ion mass,
+    and :math:`m_e` is the electron mass).
 
-    z_mean : ~astropy.units.Quantity, optional
-        The average ionization (arithmetic mean) for a plasma where the
-        a macroscopic description is valid. If this quantity is not
-        given then the atomic charge state (integer) of the ion
-        is used. This is effectively an average Alfven speed for the
-        plasma where multiple charge states are present.
+    **Aliases:** `va_`
+
+    Parameters
+    ----------
+    B : ~~astropy.units.Quantity~`
+        The magnetic field magnitude in units convertible to :math:`Tesla`.
+
+    density : ~~astropy.units.Quantity`
```

Comment:
```suggestion
    B : `~astropy.units.Quantity~`
        The magnetic field magnitude in units convertible to :math:`Tesla`.

    density : `~astropy.units.Quantity`
```
Unless there's a reason for the `~~` that I'm unaware of.

Reply:
No, not intentional.  Towards the end of last night I had noticed I started hitting `~` instead of `` ` ``.  I thought I caught all of them, but I guess not.